dev/doc/build-system.dune.md

```diff
@@ -84,6 +89,10 @@ Note that for a fast developer build of ML files, the `check` target
 is faster, as it doesn't link the binaries and uses the non-optimizing
 compiler.
 
+If you have built the full standard library with the `world` target,
+then you should be able to run the commands from the
```

Comment:
```suggestion
If you built the full standard library with the `world` target,
then you can run the commands in the
```

"from" suggests you need to `cd` to those directories

Reply:
This should be made more precise, you can run those commands always, even without a prelude if you wanna use -noinit, once #14059 is merged Coq should be able to locate the prelude in all cases, so as long as you build the prelude in any way, it would work. Other commands such as coqwc etc... for example don't need the prelude to be built to be run normally.

All in all, managing deps in a complex system like Coq by hand is super tricky, our best bet is make the build system aware of them.